docker/Dockerfile_ubuntu

```diff
@@ -53,7 +55,7 @@ RUN cmake -DCMAKE_BUILD_TYPE=Release \
         -DALICEVISION_BUILD_DOC:BOOL=OFF \
         -DALICEVISION_BUILD_EXAMPLES:BOOL=OFF \
         "${AV_DEV}" && \
-    make install -j"$(nproc)" && \
+    make install -j$((${CPU_CORES}/2)) && \
```

Comment:
I also have no problem with this. But, the reason why I put 1GB per process in check-cpu.sh was that when building dependencies, I am providing 1GB per process, because I didn't notice any crashes when building dependencies, but only during actually building AliceVision itself.

Also, if we build dependencies with a buffer of 2GB per CPU core, the dependencies can take time to build.

If it is okay then I can make the change.

Reply:
On a closer look, 2GB per process seems fine. This is because, in the worst case (that is if no of processes are lower than available memory), the system anyhow builds with nproc/2, which is the default parameters that cmake builds with. 

c2c6e25855bb8f929864a41ffa9a4635ad28e18 implements the change.